code/controllers/configuration/entries/game_options.dm

```diff
@@ -380,6 +380,11 @@
 
 /datum/config_entry/flag/shift_time_realtime
 
+/datum/config_entry/number/shift_time_start_hour
+	default = 7
```

Comment:
Actually this should default to `12` in code to match old behavior (if you had both configs disabled `gametime_offset` defaulted to `432000` or 12:00(noon)) This was a supported option and should be preserved in the config-not-in-the-txt defaults.

Keeping it 7am in the config.txt is fine, this is how we handle different defaults for new servers/forks vs existing servers.

Reply:
Got it!

Reply:
```suggestion
	default = 12
```